pkg/agent/secondarynetwork/ipam/testing/mock_ipam.go

```diff
@@ -21,7 +21,7 @@ package testing
 
 import (
 	invoke "github.com/containernetworking/cni/pkg/invoke"
-	current "github.com/containernetworking/cni/pkg/types/current"
+	types100 "github.com/containernetworking/cni/pkg/types/100"
```

Comment:
Seems you name `"github.com/containernetworking/cni/pkg/types/100"` as `current` in other files, better to keep the name consistent.

Reply:
This file is auto-generated, I will try to update it.

Reply:
OK, I didn't notice it's auto-generated file. ignore me.